Antidote/ChatViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ChatViewController.swift
+//  Antidote
+//
+//  Created by Yoba on 02/01/2017.
+//  Copyright © 2017 Dmytro Vorobiov. All rights reserved.
+//
+
+import Cocoa
+
+class ChatViewController: NSViewController {
+    let chatView = ChatView()
+    var friend: OCTFriend
+    
+    init(friendObject friend: OCTFriend) {
+        self.friend = friend
+        
+        super.init(nibName: nil, bundle: nil)!
+    }
+    
+    required init?(coder: NSCoder) {
+        self.friend = OCTFriend()
```

Comment:
This is not required. Actually this init method won't be ever used as there is not Interface Builder. Whole method can be replaced with

```swift
required init?(coder: NSCoder) {
    fatalError("init(coder:) has not been implemented")
}
```

Same applies to all other controllers/views.